github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-periodics.yaml

```diff
@@ -951,69 +765,6 @@ periodics:
     - name: pgp-bot-key
       secret:
         secretName: pgp-bot-key
-- annotations:
-    testgrid-dashboards: kubevirt-periodics
-    testgrid-days-of-results: "60"
-  cluster: prow-workloads
-  cron: 40 7,15,23 * * *
-  decorate: true
-  decoration_config:
-    grace_period: 5m0s
-    timeout: 4h0m0s
-  extra_refs:
-  - base_ref: main
-    org: kubevirt
-    repo: kubevirt
-  labels:
-    ci.kubevirt.io/prometheus: ""
-    preset-bazel-cache: "true"
-    preset-bazel-unnested: "true"
-    preset-dind-enabled: "true"
-    preset-docker-mirror-proxy: "true"
-  name: periodic-kubevirt-e2e-k8s-1.21-sig-performance
```

Comment:
I am not sure if we want to remove this one, rather bump it to use 1.24

Reply:
We have periodic-kubevirt-e2e-k8s-1.22-sig-performance-realtime which looks like the same, but just with `KUBEVIRT_REALTIME_SCHEDULER`